packages/block-editor/src/components/block-list/moving-animation.js

```diff
@@ -43,7 +43,9 @@ function useMovingAnimation( ref, isSelected, enableAnimation, triggerAnimationO
 			x: previous ? previous.left - destination.left : 0,
 			y: previous ? previous.top - destination.top : 0,
 		};
-		ref.current.style.transform = `translate3d(${ newTransform.x }px,${ newTransform.y }px,0)`;
+		ref.current.style.transform = newTransform.x === 0 && newTransform.y === 0 ?
+			undefined :
+			`translate3d(${ newTransform.x }px,${ newTransform.y }px,0)`;
```

Comment:
I wonder why applying a translate3d with all coordinates as 0 causes this problem, it seems unexpected. Supposedly translate3d(0,0,0) is a common "hack" to force hardware acceleration and makes CSS transitions smoother, but I did not expect a visual impact.

Reply:
Yes, I don't know it seems to mess up with z-index somehow.